core/templates/dev/head/services/StateTopAnswersStatsServiceSpec.ts

```diff
@@ -34,6 +42,27 @@ require('App.ts');
 require('pages/exploration-editor-page/services/exploration-states.service.ts');
 require('services/StateTopAnswersStatsService.ts');
 
+class MockAnswerStats {
+  answer: any;
```

Comment:
Where are the comments for this?

Reply:
Actually, this is in AngularJS, therefore I did not add comments.
I'll add them too.

Reply:
Ah -- I think AngularJS/Angular is the same as far as TypeScript is concerned, so yep, please add the necessary todos/comments for any code you touch.